### PR TITLE
Adding coveragerc

### PR DESCRIPTION
**Pull request by josenavas**
@ElDeveloper do you mind taking a quick look at the coveragerc file? I think you're also familiar with this file.

@ekopylova @antgonza @wasade probably you can also take a look since you're more familiar with the codebase in this repository.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# this file is based on the examples provided on scikit-learn's .coveragerc
+
+[run]
+omit =
+    */test*
+    */__init__.py
+source = benchmark,distance-method,phylogeny
+branch = True
+incldue = */*
```
Comment (wasade) [inline .coveragerc]: include

**Comment by wasade** [comment]
build is failing due to a typo in the [config](https://travis-ci.org/biocore/WGS-HGT/jobs/158878645#L2508), :+1: otherwise. btw, I'm about as familiar with the code base as you are :)

**Comment by ElDeveloper** [comment]
Looks good, oddly enough adding the configuration file dropped the coverage for 4.9% ... how is it that there was a coverage history without a coverage file 😕 ? I think this is failing due to the coverage drop.

**Comment by josenavas** [comment]
Yeah - I was also surprised by that - I would expect the coverage to go up but not sure. Is there anything blocking this PR?

**Pull request merged**